tests/versioned/apollo-server-lambda/apollo-server-lambda-setup.js

```diff
@@ -33,6 +33,7 @@ function setupApolloServerLambdaTests({suiteName, createTests, pluginConfig}, co
 
     t.beforeEach((t) => {
       const { ApolloServer, gql } = require('apollo-server-lambda')
+      const { version } = require('apollo-server-lambda/package')
```

Comment:
getting the package version to inform the helpers whether the handler should be callback or promise-based